decidim-meetings/app/views/decidim/meetings/meetings/_meetings.html.erb

```diff
@@ -1,3 +1,5 @@
+<% puts "NUM MEETINGS: #{meetings.size}" %>
```

Comment:
Hi @tramuntanal , I think this puts should not be here. Is that correct?

Reply:
solved!